src/modules/private-share-folder/private-sharing-folder.model.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  BelongsTo,
+  Column,
+  DataType,
+  ForeignKey,
+  Model,
+  PrimaryKey,
+  Table,
+} from 'sequelize-typescript';
+import { FolderModel } from '../folder/folder.model';
+import { UserModel } from '../user/user.model';
+import { PrivateSharingFolderAttributes } from './private-sharing-folder.domain';
+
+@Table({
+  underscored: true,
+  timestamps: true,
+  tableName: 'private_sharing_folder',
+})
+export class PrivateSharingFolderModel
+  extends Model
+  implements PrivateSharingFolderAttributes
+{
+  @PrimaryKey
+  @Column({ type: DataType.UUID, defaultValue: DataType.UUIDV4 })
+  id: string;
+
+  @ForeignKey(() => FolderModel)
+  @Column({ type: DataType.INTEGER, field: 'folder_id' })
+  folderId: number;
```

Comment:
This should not be used

Reply:
I was working on removing it but it brought me some problems with the join, I already have the changes in local, I finish fixing some details and I will push it in a few minutes.